src/components/v5/common/ActionSidebar/partials/ManageReputationTable/ManageReputationTable.tsx

```diff
@@ -0,0 +1,137 @@
+import clsx from 'clsx';
+import React, { type FC } from 'react';
+
+import { useMobile } from '~hooks';
+import Numeral from '~shared/Numeral/index.ts';
+import { SpinnerLoader } from '~shared/Preloaders/index.ts';
+import { formatText } from '~utils/intl.ts';
+
+import { type ManageReputationTableProps } from './types.ts';
+
+const ManageReputationTable: FC<ManageReputationTableProps> = ({
+  changeContent,
+  formattedNewReputationPoints,
+  formattedReputationPoints,
+  newPercentageReputation,
+  percentageReputation,
+  isLoading,
+  className,
+}) => {
+  const isMobile = useMobile();
+
+  const currentReputationContent = isLoading ? (
+    <SpinnerLoader />
+  ) : (
+    <p className="text-md font-normal text-gray-400">
+      <Numeral
+        value={formattedReputationPoints}
+        suffix={formatText({ id: 'actionSidebar.manageReputation.points' })}
+      />{' '}
+      (
+      <Numeral value={percentageReputation} suffix="%" trimSpaceBeforeSuffix />)
+    </p>
+  );
+
+  const newReputationContent = (
+    <p className="text-md font-normal text-gray-400">
+      <Numeral
+        value={formattedNewReputationPoints}
+        suffix={formatText({ id: 'actionSidebar.manageReputation.points' })}
+      />{' '}
+      (
+      <Numeral
+        value={newPercentageReputation}
+        suffix="%"
+        trimSpaceBeforeSuffix
+      />
+      )
+    </p>
+  );
+
+  return (
+    <div className={clsx(className, 'w-full')}>
+      <h3 className="mb-3 text-gray-900 text-2">
+        {formatText({ id: 'actionSidebar.manageReputation.reputationChange' })}
+      </h3>
+      <div className="w-full rounded-lg overflow-hidden border border-gray-200">
+        {isMobile ? (
+          <table className="w-full table-fixed">
```

Comment:
any reason why we aren't using our existing table component here? 
if yes, maybe we could at least try to reuse the styles somewhat, maybe put them into some variables